recipes/helm-rage

```diff
@@ -0,0 +1,3 @@
+(helm-rage :repo "bomgar/helm-rage"
+              :fetcher github
+              :files ("meme" "helm-rage.el"))
```

Comment:
Please use `:defaults` instead of `helm-rage.el`. Please read https://github.com/melpa/melpa#recipe-format